docs/src/main/sphinx/release/release-460.md

```diff
@@ -0,0 +1,52 @@
+# Release 460 (2 Oct 2024)
+
+## Atop connector
+
+* {{breaking}} Remove the Atop connector. ({issue}`23550`)
+
+## ClickHouse connector
+
+* Improve performance of listing columns. ({issue}`23429`)
+* Improve performance for queries comparing `varchar` columns. ({issue}`23558`) 
+* Improve performance for queries using `varchar` columns for `IN` comparisons. ({issue}`23581`)
+* Improve performance for queries with complex expression for `LIKE` clauses. ({issue}`23591`)
+
+## Delta Lake connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Add support for WASBS to [](/object-storage/file-system-azure). ({issue}`23548`)
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
+
+## Hive connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Add support for WASBS to [](/object-storage/file-system-azure). ({issue}`23548`)
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
+
+## Hudi connector
+
+* Add support for WASBS to [](/object-storage/file-system-azure). ({issue}`23548`)
+
+## Iceberg connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Add support for WASBS to [](/object-storage/file-system-azure). ({issue}`23548`)
+* Ensure table columns are cached in Glue even when table comment is too long. ({issue}`23483`)
+* Reduce planning time for queries on tables with a large number of structural
```

Comment:
Reduce planning time for queries on columns containing a large number of nested fields.